cmd/runtimetest/main.go

```diff
@@ -713,6 +721,9 @@ func run(context *cli.Context) error {
 	}
 
 	platform := runtime.GOOS
+	if err := validatePlatform(platform); err != nil {
```

Comment:
This can be moved before `loadSpecConfig`.  Non need to hit the disk at all for unsupported platforms.

Reply:
updated. PTAL.